mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -99,10 +104,16 @@ class _ChatRequest(pydantic.BaseModel, extra="forbid"):
 
 class _ChatChoice(pydantic.BaseModel, extra="forbid"):
     index: int
-    message: _ChatMessage
+    message: _ChatMessage = None
     finish_reason: Optional[str] = None
 
 
+class _ChatChoiceDelta(pydantic.BaseModel):
+    index: int
+    finish_reason: Optional[str] = None
+    delta: _ChatDeltaMessage = None
```

Comment:
```suggestion
    delta: _ChatDeltaMessage
```

it's required. None can be removed.